tests/accordion-test.ts

```diff
@@ -0,0 +1,275 @@
+        <aalam-accordion ${attrs}>
+            ${items.map(
+                (item) => html`
+                    <div class="container ${item.active ? "acc-active" : ""}">
+                        <div class="acc-title">
+                            <span>${item.title}</span>
+                            <button>Toggle</button>
+                        </div>
+                        <div class="acc-body">${item.body}</div>
+                    </div>
+                `
+            )}
+        </aalam-accordion>
+    `;
+};
+
+describe("AalamAccordion", () => {
+    it("initially opens items with the acc-active class", async () => {
+        const el = await fixture(
+            createAccordionTemplate({ activecls: "acc-active" }, [
+                {
+                    title: "What is Artificial Intelligence?",
+                    body: "body 1",
+                    active: true,
+                },
+                {
+                    title: "What is Machine Learning?",
+                    body: "body 2",
+                    active: true,
+                },
+                { title: "title 3?", body: "body 3", active: false },
+                { title: "title 4?", body: "body 4", active: false },
+                { title: "title 5?", body: "body 5", active: false },
+            ])
+        );
+
+        expect(el.querySelectorAll(".container.acc-active").length).to.equal(2);
+
+        const activeItems = el.querySelectorAll(".acc-active");
+        activeItems.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        const inactiveItems = el.querySelectorAll(
+            ".container:not(.acc-active) .acc-body"
+        );
+        inactiveItems.forEach((item) => {
+            expect(getComputedStyle(item).display).to.equal("none");
+        });
+    });
+
+    it("allows multiple items to be open if nomultiple is false", async () => {
+        const el = await fixture(
+            createAccordionTemplate(
+                { activecls: "acc-active", nomultiple: false },
+                [
+                    {
+                        title: "What is Artificial Intelligence?",
+                        body: "body 1",
+                        active: true,
+                    },
+                    {
+                        title: "What is Machine Learning?",
+                        body: "body 2",
+                        active: false,
+                    },
+                    { title: "title 3?", body: "body 3", active: false },
+                ]
+            )
+        );
+
+        const titles = el.querySelectorAll(".acc-title");
+        titles[0].click();
+        titles[1].click();
+        titles[2].click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(2);
+        expect(titles[1].parentElement?.classList.contains("acc-active")).to.be
```

Comment:
Check if only the class name is set is not enough as the class name is set by the application itself.. Its better to check the display style as thats what we do in our component.